src/py42/modules/detectionlists.py

```diff
@@ -1,7 +1,58 @@
 class DetectionListsModule(object):
     def __init__(self, microservice_client_factory):
         self._microservice_client_factory = microservice_client_factory
+        self._detection_list_user_client = None
+        # TODO Fix
+        # As this below code is failing, had to add a check in all the methods.
+        # self._detection_list_user_client = (
+        #     self._microservice_client_factory.get_detection_list_user_client()
+        # )
 
     @property
     def departing_employee(self):
         return self._microservice_client_factory.get_departing_employee_client()
+
+    @property
+    def high_risk_employee(self):
+        return self._microservice_client_factory.get_high_risk_employee_client()
+
+    def update_notes(self, user_id, notes):
+        # TODO Refactor, this condition should not be needed, relates to TODO fix above
+        if not self._detection_list_user_client:
```

Comment:
You don't need these if statements (or the private `_detection_list_user_client` and can simply get the client using `get_detection_list_user_client` in each method because the client only gets created once in that method.

Reply:
Removed.